apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.js

```diff
@@ -224,8 +224,15 @@ router.post(
         );
       }
       res.redirect(req.originalUrl);
+    } else if (req.body.__action === 'break_variants') {
+      await sqldb.queryAsync(sql.mark_variant_broken, {
+        assessment_instance_id: res.locals.assessment_instance.id,
+        instance_question_id: req.body.instance_question_id,
+        authn_user_id: res.locals.authn_user.user_id,
+      });
```

Comment:
This is all fine, but we could also do it the "modern" way by adding `breakVariantsForInstanceQuestion()` and `breakVariantsForAssessmentQuestion()` functions to `models/variants`.

Reply:
I moved this functionality to `models/variant.ts` in 7e0b69c and 66edad4.